command/v6/delete_orphaned_routes_command.go

```diff
@@ -1,18 +1,16 @@
 package v6
 
 import (
-	"code.cloudfoundry.org/cli/actor/actionerror"
 	"code.cloudfoundry.org/cli/actor/sharedaction"
 	"code.cloudfoundry.org/cli/actor/v2action"
 	"code.cloudfoundry.org/cli/command"
 	"code.cloudfoundry.org/cli/command/v6/shared"
 )
 
-//go:generate counterfeiter . DeleteOrphanedRoutesActor
+//go:generate counterfeiter . DeleteUnmappedRoutesActor
 
-type DeleteOrphanedRoutesActor interface {
-	GetOrphanedRoutesBySpace(spaceGUID string) ([]v2action.Route, v2action.Warnings, error)
-	DeleteRoute(routeGUID string) (v2action.Warnings, error)
+type DeleteUnmappedRoutesActor interface {
```

Comment:
Our naming convention for individual commands' interfaces is `NameOfCommandActor`, so this should be changed back to DeleteOrphanedRoutesActor. 

(The discrepancy between the CLI using "orphaned" and the API using "unmapped" is unfortunate)